client/app/dedicated/server/dashboard/DASHBOARD_VIEW.html

```diff
@@ -524,7 +524,8 @@
                                     <oui-action-menu-item
                                         data-text="{{:: 'server_bandwidth_order_button' | translate}}"
                                         aria-label="{{:: 'server_bandwidth_order_button' | translate}}"
-                                        data-on-click="setAction('bandwidth/order/dedicated-server-bandwidth-order', server.name)"
+                                        data-on-click="isFullAgora(server.commercialRange) === true ? setAction('bandwidth/order-agora/dedicated-server-bandwidth-agora-order', { bandwidth })
+                                            : setAction('bandwidth/order/dedicated-server-bandwidth-order', server.name)"
```

Comment:
Could the `layout: modal` be used, therefore you could have a modal with a state, at least for the agora order

Reply:
This cannot be done now as i am proceeding with `data-wizard`. This doesnt work with modal as a route. I tried